src/common/meta/src/instruction.rs

```diff
@@ -217,7 +217,9 @@ pub enum Instruction {
     /// Invalidates batch cache.
     InvalidateCaches(Vec<CacheIdent>),
     /// Flushes regions.
-    FlushRegion(FlushRegions),
+    FlushRegions(FlushRegions),
+    /// Flushes a single region.
+    FlushRegion(RegionId),
```

Comment:
Why do we need another instruction for flushing a single region provided that there's already an instruction for flushing multiple (including one of course) regions?

Reply:
FlushRegions works asynchronously and serves as a non-mandatory hint, while FlushRegion enforces a synchronous flush operation. We will later rename FlushRegions to FlushRegionsHint or FlushRegionsSuggestion to make its purpose clearer.